src/zenml/cli/base.py

```diff
@@ -475,7 +482,24 @@ def go() -> None:
             zenml_go_notebook_tutorial_message(ipynb_files), width=80
         )
         input("Press ENTER to continue...")
-    subprocess.check_call(["jupyter", "notebook"], cwd=zenml_tutorial_path)
+
+    if is_jupyter_installed():
+        try:
+            subprocess.check_call(
+                ["jupyter", "notebook"], cwd=zenml_tutorial_path
+            )
+        except subprocess.CalledProcessError as e:
+            cli_utils.error(
+                "An error occurred while launching Jupyter Notebook. "
+                "Please make sure Jupyter is properly installed and try again."
+            )
+            raise e
```

Comment:
I think this code is never reached, since `error` does this inside: `raise click.ClickException(message=click.style(text, fg="red", bold=True))`

Reply:
Correct, but if I leave it out then mypy complains. and same with the weird `e` in the docstring. If I leave that out, then `darglint` complains.